sematic/future.py

```diff
@@ -8,7 +9,12 @@
 from sematic.resolvers.resource_requirements import ResourceRequirements
 from sematic.resolvers.silent_resolver import SilentResolver
 
-_MUTABLE_FIELDS = {"name", "inline", "cache", "resource_requirements", "tags"}
+_MUTABLE_FIELDS = {"name", "standalone", "cache", "resource_requirements", "tags"}
+
+INLINE_DEPRECATION_MESSAGE = (
+    "The inline argument to the @sematic.func decorator will be deprecated on "
+    "<date>. Please use standalone=True for standalone=True."
```

Comment:
```suggestion
    "The inline argument to the @sematic.func decorator will be deprecated in "
    "the 0.31.0 release. Please use standalone=True for standalone=True."
```

Reply:
I think for these kinds of deprecation, we may want to leave a longer window open, e.g. until June 1st.